distributed_slam_karto/Graph graph
---
bool sync